commands/Moderation/mute.js

```diff
@@ -0,0 +1,35 @@
+exports.run = async (client, msg, [user]) => {
+  const { muteRole } = msg.guild.settings;
+  const value = user.roles.find("name", muteRole);
+  if (value) {
+    user.removeRole(msg.guild.roles.find("name", muteRole));
+  } else {
+    user.addRole(msg.guild.roles.find("name", muteRole));
+  }
+  msg.reply(value ? `${user} is is no longer in the time-out corner. :smiley:` : `${user} is now in the time-out corner. :smiley:`);
```

Comment:
Instead of using the emoji's representation in text, use unicode instead. You can do that by writting `\:smiley:`. Also, the message is a bit informal (it made me check the commands name).

Besides that, put a `return`.